cache.go

```diff
@@ -120,7 +120,6 @@ func (v *Validate) extractStructCache(current reflect.Value, sName string) *cStr
 	var fld reflect.StructField
 	var tag string
 	var customName string
-	
```

Comment:
Did you editor introduce this diff accidentally?

Reply:
Possibly. I believe it's gone in the latest commit though.